cgi-bin/DW/Media.pm

```diff
@@ -217,5 +217,38 @@ sub get_active_for_user {
     return sort { $b->logtime <=> $a->logtime } @media;
 }
 
+sub get_quota_for_user {
+    my ( $class, $u ) = @_;
+    confess 'Invalid user' unless LJ::isu( $u );
+
+    my $cap = $u->get_cap( 'media_file_quota' ) // 0;
+
+    # convert megabytes -> bytes
+    return $cap * 1024 * 1024;
+}
+
+sub get_usage_for_user {
+    my ( $class, $u ) = @_;
+    confess 'Invalid user' unless LJ::isu( $u );
+
+    my ( $usage ) = $u->selectrow_array(
```

Comment:
We might need to memcache this at some point ... but maybe not? Since it's just an upload check?